examples/chat.rs

```diff
@@ -0,0 +1,256 @@
+// Copyright 2021 MaidSafe.net limited.
+//
+// This SAFE Network Software is licensed to you under the MIT license <LICENSE-MIT
+// http://opensource.org/licenses/MIT> or the Modified BSD license <LICENSE-BSD
+// https://opensource.org/licenses/BSD-3-Clause>, at your option. This file may not be copied,
+// modified, or distributed except according to those terms. Please review the Licences for the
+// specific language governing permissions and limitations relating to use of the SAFE Network
+// Software.
+
+//! Basic chat like example that demonstrates how to connect with peers and exchange data.
```

Comment:
I would add a note here that this example is P2P and does NOT use/require the bootstrap server example.